.github/workflows/deploy-custom-test.yml

```
@@ -119,6 +119,7 @@ jobs:
             job_files: >-
               job_spec/INSAR_GAMMA.yml
               job_spec/INSAR_ISCE_BURST.yml
+              job_spec/VOLCSARVATORY_MULTI_BURST.yml
```

```suggestion
              job_spec/VOLCSARVATORY_MULTI_BURST.yml
```

Don't think we need two version of the multiburst job in this deployment.